packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx

```diff
@@ -31,75 +37,82 @@ export interface BasicNavigationWidgetProps {
 export function BasicNavigationWidget({
   config,
 }: BasicNavigationWidgetProps): JSX.Element {
-  const getHorizontalToolbarItems = React.useCallback((): CommonToolbarItem[] => {
-    if (config?.hideDefaultHorizontalItems) {
-      return [];
-    }
-    const items: CommonToolbarItem[] = [];
+  const getHorizontalToolbarItems =
```

Comment:
are these changes becasue of the changes you made to formatting?

Reply:
yeah i just did a shift alt f and I'm pretty sure this is what changed, should I undo the changes?